.github/workflows/main.yml

```
@@ -0,0 +1,65 @@
+name: Main
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  check-workflows:
+    name: Check workflows
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Download actionlint
+        id: download-actionlint
+        run: bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/7fdc9630cc360ea1a469eed64ac6d78caeda1234/scripts/download-actionlint.bash) 1.6.23
```

should this be in silent mode?

```suggestion
        run: bash <(curl  -s https://raw.githubusercontent.com/rhysd/actionlint/7fdc9630cc360ea1a469eed64ac6d78caeda1234/scripts/download-actionlint.bash) 1.6.23
```

---

not sure, this is copy-pasted from: https://github.com/MetaMask/metamask-module-template

---

Thats ok then, I'd have added the silent flag, but that's just a personal preference